api/orchestrator/orchestrator.proto

```diff
@@ -655,16 +655,20 @@ message CloudService {
     (validate.rules).repeated .items.message.required = true
   ];
 
-  // As long as repeated maps are not supported, we have to do this workaround. 
-  message Tag {
-    map<string, string> tag   = 1;
+  message Metadata {
+    // a map of key/value pairs, e.g., env:prod
+   map<string, string> labels = 1 ;
+   
+   // an icon for the cloud service used by the UI
+   string icon = 2;
+  
+   // a color for the cloud service used by the UI
+   string color = 3;
```

Comment:
The color was part of the catalog, not the cloud service. We can probably re-use the same metadata message for catalog and cloud service?